upgrade/v2.5/script/upgrade.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+
+# Set the ENV var for KDD backend
+export DATASTORE_TYPE=kubernetes
+
+# Create the CRDs by applying the crds.yaml manifest.
+echo "Creating Calico CRDs..."; echo
+/sbin/kubectl apply -f crds.yaml
+# Applying TPRs with trps.yaml manifest.
+/sbin/kubectl apply -f tprs.yaml
+
+# Get IPPools list with old calicoctl (from TPR) and save it as a yaml file.
+# Apply the yaml file with the new calicoctl (to CRD)
+echo "Migrating IPPools..."
+/sbin/calicoctl-v1.4 get ippool -o yaml > ippool.yaml
+if [ $? != 0 ]; then
```

Comment:
As Brendan pointed out the equality and inequality operators used in the script are the string versions instead of the numerical versions, but I think all of the comparisons will function as expected though.  I don't expect them to be changed but wanted to make note that it was something I specifically reviewed.